backend/bamboo/utils.py

```diff
@@ -1,6 +1,101 @@
-from datetime import UTC, datetime
+import uuid
+from datetime import UTC, datetime, timedelta
+from typing import (
+    Any,
+    Container,
+    Iterable,
+    MutableMapping,
+)
+
+from jose import jwt
 
 
 def utc_now() -> datetime:
     """Get the current time in UTC."""
     return datetime.now(UTC)
+
+
+def _encode_jwt(
```

Comment:
Why making it prefixed with underscore? Linters might emit warnings about this when used by other modules

Reply:
Because it's an internal function instead of a public function. 
But It doesn't matter in this app. It's a web app, not a library.
We can remove the underscore.

Reply:
internal function means it can be only called by the other code **in the same module**, but I don't this is the case right?

Reply:
Sorry! It's a mistake.